README.md

```diff
@@ -1,2 +1,180 @@
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3.org/html/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">SCSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.netlify.com/">Netlify</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Linters**: [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/), [Webhint](https://webhint.io/), and [Stylelint](https://stylelint.io/.)
+- **A `.github/workflows/linters.yml`** file to run linters on GitHub Pull Request.
+- **A** `.gitignore` file, including `node_modules/`.
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://hello-microverse.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> If you are a new developer and would like to have a basic project boilerplate to start practicing your code or creating your first app-features, cloning this project will give you what you need!
+> To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Have [npm](https://www.npmjs.com/package/npm) installed in your project:
+  ```sh
+  curl -qL https://www.npmjs.com/install.sh | sh
+  ```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+git clone https://github.com/astroboyReloaded/Hello-Microverse.git
+```
+
+or
+
+`git clone git@github.com:astroboyReloaded/Hello-Microverse.git` <-- _Only if you have [SSH keys configured](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) on your GitHub account_.
+
+### Install:
+
+run the 
+```sh
+npm install` command in your console.
+```
+
+### Usage
+
+To run the linters before committing any changes to your code, execute the following commands:
+
+```sh
+lighthouse <url>
+```
+```sh
+npx hint .
+```
+```sh
+`npx stylelint "**/*.{css,scss}"`
+``` 
+_NOTE: this repo is set up to use with CSS. If you would like to use [SCSS](https://sass-lang.com/)., you must replace `"**/*.{css,scss}"` with `"**/*.scss"` in **line 48** of the `linters.yml` file, and run the linter with:_
+```sh
+npx stylelint "**/*.scss"
+```
+ _instead_.
+ 
+ ```sh
+ npx eslint .
+ ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> 👤 **AstroboyReloaded**
+
+- GitHub: [@astroboyReloaded](https://github.com/astroboyReloaded)
+- Twitter: [@astroboyReload](https://twitter.com/astroboyReload)
+- LinkedIn: [Alex Muñoz](https://www.linkedin.com/in/astroboyreloaded/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ESLint](https://eslint.org/).
+
```

Comment:
- [OPTIONAL] You can think of better future features that you're most likely planning to add in future.